### DISCUSSION TITLE
Manually print drawing in Jupyter notebook?

### DISCUSSION THREAD

**Discussion by iyanmv** [cduck/drawSvg#74] [Q&A] [answered]
Hi,

How can I manually print a drawing in a cell? I need this so I can print multiple drawings in the same cell.

For example, this works

```python
d1 = draw.Drawing(100, 100)
d1.append(draw.Circle(25, 50, 20))
d1
```

But this doesn't

```python
d1 = draw.Drawing(100, 100)
d1.append(draw.Circle(25, 50, 20))
d1
<some other code>
d2 = draw.Drawing(100, 100)
d2.append(draw.Circle(25, 50, 20, fill='red'))
d2
```

Only `d2` is shown.

I have tried with `displayInline=True` and `displayInline=False`, using the `asSvg()` method, also `_repr_html_()` and `_repr_svg_`. But nothing seems to do what I need.

Any ideas?

Also, what is exactly the purpose of `displayInline`, doesn't seem to make any difference to me (maybe it only works on Jupyter Notebook? I'm using Jupyter Lab)

**Comment by iyanmv** [comment, marked as answer]
Okay, I thing I got it now. Using `display` and `SVG` from `IPython.display`

```python
d1 = draw.Drawing(100, 100, displayInline=False)
d1.append(draw.Circle(25, 50, 20))

d2 = draw.Drawing(100, 100)
d2.append(draw.Circle(25, 50, 20, fill='red'))

display(SVG(d1.asSvg()), SVG(d2.asSvg()))
```

**Reply by cduck** [reply]
Your solution is very close to the solution I use:

```python
from IPython.display import display

d1 = draw.Drawing(100, 100, displayInline=False)
d1.append(draw.Circle(25, 50, 20))
display(d1)

d2 = draw.Drawing(100, 100)
d2.append(draw.Circle(25, 50, 20, fill='red'))
display(d2)
```

You don't need to wrap the SVG content in `SVG()` because `Drawing` objects support Jupyter's rich repr API.

**Reply by iyanmv** [reply]
Oh, nice! Thanks
